ext/standard/tests/math/is_finite_basic.phpt

```diff
@@ -2,26 +2,29 @@
 Test is_finite() - basic function test is_finite()
 --FILE--
 <?php
-$values = array(234,
-                -234,
-                23.45e1,
-                -23.45e1,
-                0xEA,
-                0352,
-                "234",
-                "234.5",
-                "23.45e1",
-                true,
-                false,
-                pow(0, -2),
-                acos(1.01));
-;
+$values = [
+    234,
+    -234,
+    23.45e1,
+    -23.45e1,
+    0xea,
+    0352,
+    "234",
+    "234.5",
+    "23.45e1",
+    true,
+    false,
+    pow(0, -2),
+    acos(1.01),
+];
 for ($i = 0; $i < count($values); $i++) {
     $res = is_finite($values[$i]);
     var_dump($res);
 }
 ?>
---EXPECT--
+--EXPECTF--
+
```

Comment:
Nit: Whitespace can be removed. I believe the output is trimmed anyway.

Reply:
Removed